Makefile

```diff
@@ -47,6 +53,13 @@ test-e2e:
 	./openshift/e2e-tests-openshift.sh
 .PHONY: test-e2e
 
+# Run make DOCKER_REPO_OVERRIDE=<your_repo> test-e2e-local if test images are available
+# in the given repository. Make sure you first build and push them there by running 
+# `make DOCKER_REPO_OVERRIDE=<your_repo> test-images`.
+test-e2e-local:
+	KO_DOCKER_REPO=$(DOCKER_REPO_OVERRIDE) ./test/local-e2e-tests.sh
```

Comment:
We'd need anyuid hack for --user e2e tests

```
  # Add anyuid scc to all authenticated users so e2e tests for --user flag can user any user id
  oc adm policy add-scc-to-group anyuid system:authenticated
```

beside, I havent tried this locally lately (after we added the image building in CI)

`KO_DOCKER_REPO=$(DOCKER_REPO_OVERRIDE) ./test/local-e2e-tests.sh`

Reply:
Those should be resolved in the latest commits.